packages/core/helper-plugin/lib/src/hooks/useTracking/index.js

```diff
@@ -5,22 +5,31 @@ import useAppInfos from '../useAppInfos';
 
 const useTracking = () => {
   const trackRef = useRef();
-  const { uuid, telemetryProperties } = useContext(TrackingContext);
+  const { uuid, telemetryProperties, deviceId } = useContext(TrackingContext);
   const appInfo = useAppInfos();
+  const adminUserId = appInfo?.adminUserId;
 
   trackRef.current = async (event, properties) => {
     if (uuid && !window.strapi.telemetryDisabled) {
       try {
-        await axios.post('https://analytics.strapi.io/track', {
-          event,
-          properties: {
-            ...telemetryProperties,
-            ...properties,
-            projectType: window.strapi.projectType,
-            environment: appInfo.currentEnvironment,
+        await axios.post(
+          'https://analytics.strapi.io/api/v2/track',
+          {
+            event,
+            adminUserId,
+            deviceId,
+            eventProperties: { ...properties },
+            userProperties: {},
+            groupProperties: {
+              ...telemetryProperties,
```

Comment:
Are we sure those are only group Properties? it looks lik eit could contain event/user & group properties eventually

Reply:
We discussed it [here](https://github.com/strapi/strapi/pull/14083#discussion_r1006919330) and I agree, but it's likely that it will involve many changes.

Reply:
the telemetry endpoint could simply return an object containing 

groupProp and here we would use a merge instead of a spread. Not sure why this would require more changes? what do you have in mind ?

Reply:
Oh, I see, you're only talking about what's coming from the context, and indeed yeah the changes should be kept minimal.

I was talking about allowing the user to send all kinds of properties when using the `useTracking` hook, cause right now we can only pass event properties.

Reply:
Yes not the same thing but this could be interesting too allow indeed. 

We can refactor that code when we will need to send other properties anyway

Reply:
I'll resolve this conversation then since as I understand no actions are currently necessary. For this time those are indeed the only `groupProperties`, but that can be refactored in the future should there be a need 😊👍

Reply:
Unless I misunderstood, I think he wanted to have a different format for the `telemetryProperties` coming from the context so that not everything simply ends up in group properties.